### PR TITLE
ocm - Only run framework tests when they're updated (GRC)

### PR DESCRIPTION
**Pull request by dhaiducek**
- https://github.com/stolostron/backlog/issues/23536

### CODE REVIEW

File: ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main.yaml [hunk 1]
```diff
@@ -73,6 +73,7 @@ tests:
           memory: 200Mi
     workflow: ocm-e2e-kind
 - as: test-e2e-grc-framework
+  run_if_changed: ^test/integration/.*
```
Comment (mprahl) [inline ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main.yaml:76]: Would this affect the periodic test runs also or just the PRs?

**Comment by mprahl** [comment]
@dhaiducek overall, I think this is a good change to not run the integration tests on PRs unless that directory was modified.

I was originally thinking about skipping only the flaky tests such as Gatekeeper, Compliance Operator, and Kyverno unless those individual test files were modified, but I think this is fine.

What do you think @JustinKuli?

File: ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main.yaml (hunk 1 above)
Comment (dhaiducek) [reply ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main.yaml:76]: Just the PRs. The periodic test runs in Travis.

**Comment by dhaiducek** [comment]
Hmmm. Good point. I suppose I did take a sledgehammer to the issue... 🔨

**Review by JustinKuli — APPROVED** [review]

**Pull request merged**